typescript/packages/subsurface-viewer/src/layers/axes2d/label-vertex.glsl.js

```diff
@@ -1,20 +1,18 @@
 export default `\
 #version 300 es
-#define SHADER_NAME graph-layer-axis-vertex-shader
+#define SHADER_NAME axes2d-layer-vertex-shader //graph-layer-axis-vertex-shader
 
 precision highp float;
 
-in  vec3 positions;
+in vec3 positions;
 
 in vec2 vTexCoord;
 out vec2 _vTexCoord;
 
-uniform mat4 projectionMatrix;
-
 void main(void) {
    _vTexCoord = vTexCoord;
 
-   vec3 position_commonspace = positions; // These positions are in view space.
-   gl_Position = projectionMatrix * vec4(position_commonspace, 1.0); // From viewspace to clip
+   vec3 position_commonspace = project_position(positions);
```

Comment:
This layer used to position all labels in view space/camera space and then the vertex shader from view spice to clip space. Now I changed it so that the labels are in world space and then the vertex shader could be reverted to the "standard" world-to-clipspace conversion. I don't think the last coordinate matters here but I changed it to 1.0.

Reply:
Thank you.
I did not realize the big change was in the layer (the diff is too big to load automatically ;)